### PR TITLE
Reorg step1

### PR DESCRIPTION
**Pull request by ry**
Ref #285

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/src/out/
+/src/js/node_modules/
```
Comment (qti3e) [inline .gitignore]: `node_modules` would work too

Comment (ry) [reply .gitignore]: Thanks, fixed.

File: src/file_util.cc [hunk 2]
```diff
@@ -78,7 +78,7 @@ class StartupDataCppWriter {
     WriteBinaryContentsAsCArray();
     file_ << "};\n";
     snprintf(buffer, sizeof(buffer), "static const int %s_blob_size = %llu;\n",
-             name_, static_cast<unsigned long long>(data_.size()));
+             name_, static_cast<uint64_t>(data_.size()));
```
Comment (piscisaureus) [inline src/file_util.cc]: Why this change? This is not correct.

Comment (piscisaureus) [reply src/file_util.cc]: I don't like unsigned long long either. But my compiler complains that "%llu" isn't appropriate for uint64_t.
The correct way to do this, then is to use the PRId64 macro. It's horrible, but it works:
`snprintf(buffer, sizeof(buffer), "static const int %s_blob_size = " PRId64 ";\n",

Comment (ry) [reply src/file_util.cc]: wow. ok. never seen PRId64 before. will fix next time.
